<launch>
  
        <node pkg="pioneer3at" type="frontier_explorer" name="Pioneer3AT_frontier_explorer" required="true">
            <param name="robot_topic" type="str" value="Pioneer3AT" />
            <param name="goal_topic" type="str" value="/Pioneer3AT/move_base_simple/goal" />
            <param name="map_topic" type="str" value="/Pioneer3AT/map" />
            <param name="pose_topic" type="str" value="/Pioneer3AT/odom" />
            <param name="base_link_topic" type="str" value="/Pioneer3AT/base_link" />
            <param name="occ_topic" type="str" value="/Pioneer3AT/occ_map" />
            <param name="goal_status_topic" type="str" value="/Pioneer3AT/move_base/status" />
        </node>

</launch>
